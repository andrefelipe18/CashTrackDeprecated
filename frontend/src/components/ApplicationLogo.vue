<template>
   <img src="/logo.svg" class="h-36" alt=""> 
</template>
