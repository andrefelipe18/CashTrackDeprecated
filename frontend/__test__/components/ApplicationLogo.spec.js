import { describe, it, expect } from 'vitest' 
import { mount } from '@vue/test-utils'
import ApplicationLogo from '../../src/components/ApplicationLogo.vue'

describe('ApplicationLogo', () => {
    
    it('should render correctly', () => { //Deve renderizar corretamente
        const wrapper = mount(ApplicationLogo);
        expect(wrapper.html()).toMatchSnapshot(); 
    })
});
//Explicação sobre o snapshot
//O snapshot é um arquivo que contém uma representação do componente no momento em que o teste foi executado.