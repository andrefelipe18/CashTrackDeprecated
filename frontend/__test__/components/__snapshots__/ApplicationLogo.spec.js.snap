// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ApplicationLogo > should render correctly 1`] = `"<img src=\\"/logo.svg\\" class=\\"h-36\\" alt=\\"\\">"`;
